README.md

```diff
@@ -1,14 +1,38 @@
 
-### Build
+## Compatibility
 
-mvn clean install
+| Connector version | Supported Salesforce SOAP API version | Supported WSO2 ESB/EI version |
+| ------------- | ------------- | ------------- |
+| [1.0.6](https://github.com/wso2-extensions/esb-connector-salesforcebulk/tree/org.wso2.carbon.connector.salesforcebulk-1.0.6) | 34.0 | ESB 5.0.0, ESB 4.9.0, EI 6.3.0, EI 6.4.0    |
+| [1.0.5](https://github.com/wso2-extensions/esb-connector-salesforcebulk/tree/org.wso2.carbon.connector.salesforcebulk-1.0.5) | 34.0 | ESB 4.9.0, 5.0.0, 6.1.0, 6.1.1 |
+| [1.0.4](https://github.com/wso2-extensions/esb-connector-salesforcebulk/tree/org.wso2.carbon.connector.salesforcebulk-1.0.4) | 34.0 | ESB 4.9.0, 5.0.0  |
 
-### How You Can Contribute
-You can create a third party connector and publish in WSO2 Store.
+## Getting started
 
-https://docs.wso2.com/display/ESBCONNECTORS/Creating+and+Publishing+a+Third+Party+Connector
\ No newline at end of file
+#### Download and install the connector
+
+1. Download the connector from [WSO2 Store](https://store.wso2.com/store/assets/esbconnector/details/3957b581-86e0-4909-ab75-c3cfa2ce6494) by clicking the Download Connector button.
+2. Then you can follow this [Documentation](https://docs.wso2.com/display/EI640/Working+with+Connectors+via+the+Management+Console) to add and enable the connector via the Management Console in your EI instance.
+3. For more information on using connectors and their operations in your EI configurations, see [Using a Connector](https://docs.wso2.com/display/EI640/Using+a+Connector).
+4. If you want to work with connectors via EI tooling, see [Working with Connectors via Tooling](https://docs.wso2.com/display/EI640/Working+with+Connectors+via+Tooling).
+
+#### Configuring the connector operations
+
+To get started with Salesforce Bulk connector and their operations, see [Configuring Salesforce Bulk Operations](docs/config.md).
+
+
+## Building From the Source
+
+If you want to build Salesforce Bulk connector from the source code:
+
+1. Get a clone or download the source from [Github](https://github.com/wso2-extensions/esb-connector-salesforcebulk).
+2. Run the following Maven command from the `esb-connector-salesforcebulk` directory: `mvn clean install`.
+3. Salesforce connector zip will be created under `esb-connector-salesforcebulk/target` directory
```

Comment:
The Salesforce connector zip file is created in the `esb-connector-salesforcebulk/target` directory.